projects/go-lib/src/lib/components/go-input/go-input.component.ts

```diff
@@ -7,14 +7,16 @@ import { FormGroup } from '@angular/forms';
 })
 export class GoInputComponent {
 
-  @Input() label: string;
-  @Input() inputType: string = 'text';
+  @Input() controlName: string;
   @Input() errors: string[];
   @Input() errorStatus: string = 'Error:';
   @Input() hints: string[];
-  @Input() controlName: string;
+  @Input() inputDisabled: boolean;
+  @Input() inputType: string = 'text';
+  @Input() label: string;
   @Input() parentFormGroup: FormGroup;
   @Input() placeholder: string = '';
+  @Input() theme: string = 'light';
 
   constructor() { }
```

Comment:
Should we throw some errors in the `constructor` if any of the required items are not passed? Just a thought here, but we could do something like:
```
constructor() {
  if (!this.controlName) { throw new Error('GoInput: controlName is a required Input'); }
}
```

Reply:
I could have done it wrong, but when I put this in the constructor it was throwing errors even when passing in the control name as it hadn't been defined yet. I went ahead and added it to the onInit tho

Reply:
@StevenUlmer, my bad, I meant `onInit`. Thanks though, I think that'll help a lot.